kde-frameworks/attica/attica6-6.5.0.recipe

```diff
@@ -70,12 +79,15 @@ INSTALL()
 
 	packageEntries devel \
 		$developDir \
-		$dataDir/doc \
+		$documentationDir \
 		$libDir/cmake
 }
 
 TEST()
 {
+	pip3 install --user reuse
```

Comment:
Can be commented out (was planning to) as I already did for another package (local), look at it more as a test case on how it could be done then rather something useful. :) (ps, this doesn't fail the testcase for attica if not available).